tests/common/platform/processes_utils.py

```diff
@@ -0,0 +1,49 @@
+"""
+Helper script for checking status of critical processes
+
+This script contains re-usable functions for checking status of critical services.
+"""
+import logging
+import time
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import wait_until
+
+
+def _get_critical_processes_status(dut):
+    processes_status = dut.all_critical_process_status()
+    for k, v in processes_status.items():
+        if v['status'] == False or len(v['exited_critical_process']) > 0:
+            return False, processes_status
+
+    return True, processes_status
+
+def _all_critical_processes_healthy(dut):
+    logging.info("Check critical processes status")
+    status, _ = _get_critical_processes_status(dut)
+    return status
+
+def check_critical_processes(dut, watch_secs=0):
```

Comment:
Why are we not using the existing 'all_critical_process_status' in devices.py?

Reply:
We are using it. See line 14 :-)

Reply:
ah I see. My bad